### DISCUSSION TITLE
Is this library still active?

### DISCUSSION THREAD

**Discussion by sotoz** [rossmann-engineering/EasyModbusTCP.NET#98] [Q&A]
Hello, 

I was wondering if people are actively using and/or contributing to this library and in general if there are plans for further development/releases etc. I see last release was done back in 2021 but I see some code changes after the last release.

Thank you in advance.

**Comment by nbkgit** [comment]
i am still using it, but with every new Dot Net version it gets harder to get it to un, then uget packe is also corrupted
